<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewGameClass</class>
 <widget class="QDialog" name="DialogNewGameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New game</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>143</width>
     <height>33</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSelectFile">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../../interface.qrc">
     <normaloff>:/images/interface/open.png</normaloff>:/images/interface/open.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFileName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPlayerWhite">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPlayerBlack">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLevelWhite">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLevelBlack">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../interface.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogNewGameClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogNewGameClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
